src/frontend/components/UI/AccountDropdown/index.tsx

```diff
@@ -50,48 +52,46 @@ export default function AccountDropdown() {
         <Menu.Label className={styles.menuLabel}>
           {t('hyperplay.currentWallet', `Current wallet`)}
         </Menu.Label>
-        {showMetaMaskBrowserSidebarLinks && (
+        {showWalletLinks && (
           <>
             <NavigationMenuItem
               label={t('hyperplay.viewFullscreen', `View fullscreen`)}
               to={'/metamaskHome'}
+              showWalletLinks={showWalletLinks}
             ></NavigationMenuItem>
             <NavigationMenuItem
               label={t('hyperplay.viewItem', {
                 defaultValue: 'View {{item}}',
                 item: 'Snaps'
               })}
+              showWalletLinks={showWalletLinks}
               to={'/metamaskSnaps'}
             ></NavigationMenuItem>
           </>
         )}
-        <Menu.Item
-          className={`${styles.menuItem} `}
-          id={
-            !showMetaMaskBrowserSidebarLinks
-              ? 'topMenuItemWalletDropdown'
-              : undefined
-          }
-        >
-          <NavLink
-            to={'/metamaskPortfolio'}
-            id={
-              !showMetaMaskBrowserSidebarLinks
-                ? 'topElementWalletDropdown'
-                : undefined
-            }
+        {showWalletLinks && (
+          <Menu.Item
+            className={`${styles.menuItem} `}
+            id={!showWalletLinks ? 'topMenuItemWalletDropdown' : undefined}
           >
-            <div className={`body ${styles.itemContents}`}>
-              {t('hyperplay.viewPortfolio', `View portfolio`)}
-            </div>
-          </NavLink>
-        </Menu.Item>
+            <NavLink
+              to={'/metamaskPortfolio'}
+              id={!showWalletLinks ? 'topElementWalletDropdown' : undefined}
+            >
+              <div className={`body ${styles.itemContents}`}>
+                {t('hyperplay.viewPortfolio', `View portfolio`)}
+              </div>
+            </NavLink>
+          </Menu.Item>
+        )}
```

Comment:
the portfolio should be shown when any wallet is connected, so you'll end up needing two flags 
```
showMetaMaskExtensionLinks = walletState.isConnected && walletState.provider === PROVIDERS.METAMASK_EXTENSION

showWalletConnectedLinks = walletState.isConnected
```

Reply:
will fix

Reply:
should be fixed.